tests/requirements/dj42_cms40.txt

```diff
@@ -0,0 +1,9 @@
+-r requirements_base.txt
+
+Django>=4.2,<5.0
+
+https://github.com/django-cms/django-cms/tarball/release/4.0.1.x#egg=django-cms
+https://github.com/django-cms/djangocms-text-ckeditor/tarball/support/4.0.x#egg=djangocms-text-ckeditor
```

Comment:
@joshyu Can you use the latest text ckeditor branch

Reply:
I will update the requirements  later. Thanks for pointing this out.